### PR TITLE
Support Bun

### PR DESCRIPTION
**Pull request by SnowMarble**
Edited to check Bun as well. But, I need to fix it after #19 is merged.

### CODE REVIEW

**Comment by FleetAdmiralJakob** [comment]
Any updates?

**Comment by tomerh2001** [comment]
+1

**Comment by tomerh2001** [comment]
Any updates?

**Comment by tomerh2001** [comment]
!19 had been merged, can this be implemented now?

**Review by zkochan — APPROVED** [review]

**Comment by tomerh2001** [comment]
@zkochan The PR has been merged but there isn't a new release on NPM(?)

P.S. I suggest checking out [Semantic Release](https://github.com/semantic-release/semantic-release)

**Comment by zkochan** [comment]
I am not going to setup semantic release for a project that is released ones in a few years.

I have published the new version.

**Comment by tomerh2001** [comment]
Has anyone actually tried it? Cause it doesn't work

File: bin.js [hunk 1]
```diff
@@ -44,6 +44,12 @@ For more details, go to https://pnpm.js.org/`))
 
 If you don't have Yarn, install it via "npm i -g yarn".
 For more details, go to https://yarnpkg.com/`))
+
```
Comment (qeleb) [inline bin.js:47]: missing:
```js
break;
```

this bugs causes it to output both yarn and bun tips when it fails

Comment (SnowMarble) [reply bin.js:47]: nice catch! I made a new PR for this issue - #25

**Pull request merged**